flottform/demo/src/routes/custom-ui/+page.svelte

```diff
@@ -0,0 +1,586 @@
+								transform=" matrix(1 0 0 1 0 0) "
+								stroke-linecap="round"
+								pathLength="1"
+								stroke-width="4"
+								stroke="#3ab53a"
+								class="second"
+							/>
+						</svg>
+						<p class="text-2xl font-handwriting typewriter">Upload your photo</p>
+					</div>
+				{/if}
+			</div>
+			<button
+				type="submit"
+				class="group relative w-fit cursor-pointer overflow-hidden rounded-md border-2 border-primary-blue px-12 py-3 font-semibold"
+			>
+				<span
+					class="ease absolute top-1/2 h-0 w-64 origin-center -translate-x-20 rotate-45 bg-primary-blue transition-all duration-300 group-hover:h-64 group-hover:-translate-y-32"
+				/>
+				<span class="ease relative transition duration-300 group-hover:text-white">Send</span>
+			</button>
+		</form>
+	</div>
+</div>
+
```

Comment:
And here we can add `dialog` element directly

```suggestion
<dialog bind:this={dialog} class="p-4 flex flex-col gap-4 justify-center items-center">
	{#if isOpenModal}
		<button
			type="button"
			on:click={() => {
				dialog.close();
				isOpenModal = !isOpenModal;
			}}
			class="self-end"
		>
			<svg
				xmlns="http://www.w3.org/2000/svg"
				width="30"
				height="30"
				viewBox="0 0 15 15"
				fill="currentColor"
			>
				<path
					fill-rule="evenodd"
					clip-rule="evenodd"
					d="M6.79289 7.49998L4.14645 4.85353L4.85355 4.14642L7.5 6.79287L10.1464 4.14642L10.8536 4.85353L8.20711 7.49998L10.8536 10.1464L10.1464 10.8535L7.5 8.20708L4.85355 10.8535L4.14645 10.1464L6.79289 7.49998Z"
					fill="currentColor"
				/>
			</svg>
		</button>
		<h2 class="">Upload a file!</h2>
		<p>Use this QR-Code or Link on your other device.</p>
		<div class="flex flex-col gap-4 items-center">
			<img src={webRtcQrCode} alt="qr code to upload file from other device" class="w-56" />
			<div>{webRtcLink}</div>
		</div>
	{/if}
</dialog>
```